modules/landing-page/landing-pricing.tsx

```
@@ -0,0 +1,46 @@
+        </h2>
+        <p className="mx-auto mb-12 max-w-2xl text-xl leading-relaxed text-gray-300">
+          Join thousands of individuals and teams who've transformed their productivity with simple,
+          powerful task management.
+        </p>
+
+        <div className="mb-12 flex flex-col items-center justify-center gap-6 sm:flex-row">
+          <button className="group flex items-center rounded-full bg-white px-10 py-5 text-lg font-bold text-black shadow-2xl transition-all duration-300 hover:bg-gray-100">
+            Start Shipping
+            <ArrowRight className="ml-3 h-5 w-5 transition-transform group-hover:translate-x-1" />
+          </button>
+          <button className="group flex items-center rounded-full border-2 border-gray-600 px-10 py-5 text-lg font-medium text-white transition-all duration-300 hover:border-white">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)